### PR TITLE
Add missing imports in Data Mappers article

### PR DESCRIPTION
**Pull request by gregurco**

### CODE REVIEW

**Review by xabbuh — APPROVED** [review]

File: form/data_mappers.rst [hunk 1]
```diff
@@ -150,6 +150,8 @@ method to configure the data mapper::
 
     use App\Form\DataMapper\ColorMapper;
     use Symfony\Component\Form\Extension\Core\Type\IntegerType;
+    use Symfony\Component\Form\FormBuilderInterface;
+    use Symfony\Component\OptionsResolver\OptionsResolver;
```
Comment (HeahDude) [inline form/data_mappers.rst]: Thanks for fixing this :). What about adding a use for the `AbstractType` too then?

Comment (gregurco) [reply form/data_mappers.rst]: Added! Thank you :slightly_smiling_face:

**Review by HeahDude — APPROVED** [review]
Perfect, thanks!

**Comment by javiereguiluz** [comment]
These imports are extremely important for us ... so thanks for adding the missing ones!

**Pull request merged**